AMDirT/core/__init__.py

```diff
@@ -23,6 +23,17 @@
 logger.propagate = False
 
 
+def monkeypatch_get_storage_manager():
+    if st.runtime.exists():
+        return st.runtime.get_instance().cache_storage_manager
+    else:
+        # When running in "raw mode", we can't access the CacheStorageManager,
```

Comment:
Link to the URL where you took that info form?

Reply:
Added the link to the issue, but this comment is litterally coming from the streamlit code itself.